docs/how_to/deploy.md

```diff
@@ -35,3 +35,84 @@ out = tvm.nd.empty(shape)
 get_output(0, out)
 print(out.asnumpy())
 ```
+
+An example in c++.
+```cpp
+#include <dlpack/dlpack.h>
+#include <tvm/runtime/module.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/packed_func.h>
+
+#include <fstream>
+#include <algorithm>
+
+int main()
+{
+    // tvm module for compiled functions
+    tvm::runtime::Module mod_dylib = tvm::runtime::Module::LoadFromFile("deploy.so");
+
+    // json graph
+    std::ifstream json_in("deploy.json", std::ios::in);
+    std::stringstream json_ss;
```

Comment:
`<sstream>` should be included for the use of `stringstream`. BTW, you can copy only once:

```cpp
std::string json_data(std::istream_iterator<char>(json_in), std::istream_iterator<char>());
```

Reply:
Please use the iterator initializer method